apps/prairielearn/src/lib/client/safe-db-types.ts

```diff
@@ -10,7 +10,7 @@ import {
 } from '../db-types.js';
 
 /** Assessments */
-export const RawStaffAssessmentSchema = RawAssessmentSchema;
+export const RawStaffAssessmentSchema = z.object(RawAssessmentSchema.shape);
```

Comment:
This process is now needed throughout this file as we _don't_ want these to be strict.